src/test/cypress/e2e/exercises/text/TextExerciseManagement.cy.ts

```diff
@@ -40,7 +40,6 @@ describe('Text exercise management', () => {
         textExerciseCreation.setDueDate(dayjs().add(1, 'days'));
         textExerciseCreation.setAssessmentDueDate(dayjs().add(2, 'days'));
         textExerciseCreation.typeMaxPoints(10);
-        textExerciseCreation.checkAutomaticAssessmentSuggestions();
```

Comment:
Note for reviewers: This does no longer make sense because the checkbox now only appears when it can actually be enabled (when the `athena` profile is active), which is not the case for the Cypress tests